circus-train-core/src/main/java/com/hotels/bdp/circustrain/CircusTrain.java

```diff
@@ -98,25 +97,27 @@ public static void main(String[] args) throws Exception {
     int exitCode = -1;
     try {
       String defaultModules = Joiner.on(",").join(Modules.REPLICATION, Modules.HOUSEKEEPING);
-      exitCode = SpringApplication.exit(new SpringApplicationBuilder(CircusTrain.class)
-          .properties("spring.config.location:${config:null}")
-          .properties("spring.profiles.active:${modules:" + defaultModules + "}")
-          .properties("instance.home:${user.home}")
-          .properties("instance.name:${source-catalog.name}_${replica-catalog.name}")
-          .properties("jasypt.encryptor.password:${password:null}")
-          .properties("housekeeping.schema-name:circus_train")
-          .registerShutdownHook(true)
-          .initializers(new ExtensionInitializer())
-          .listeners(new ConfigFileValidationApplicationListener())
-          .build()
-          .run(args));
+      exitCode = SpringApplication
+          .exit(new SpringApplicationBuilder(CircusTrain.class)
+              .properties("spring.config.location:${config:null}")
+              .properties("spring.profiles.active:${modules:" + defaultModules + "}")
+              .properties("instance.home:${user.home}")
+              .properties("instance.name:${source-catalog.name}_${replica-catalog.name}")
+              .properties("jasypt.encryptor.password:${password:null}")
+              .properties("housekeeping.schema-name:circus_train")
+              .registerShutdownHook(true)
+              .initializers(new ExtensionInitializer())
+              .listeners(new ConfigFileValidationApplicationListener())
+              .build()
+              .run(args));
     } catch (ConfigFileValidationException e) {
       LOG.error(e.getMessage(), e);
       printCircusTrainHelp(e.getErrors());
     } catch (BeanCreationException e) {
       LOG.error(e.getMessage(), e);
-      if (e.getMostSpecificCause() instanceof BindException) {
-        printCircusTrainHelp(((BindException) e.getMostSpecificCause()).getAllErrors());
+      Throwable mostSpecificCause = e.getMostSpecificCause();
+      if (mostSpecificCause instanceof BindException) {
+        printCircusTrainHelp(((BindException) mostSpecificCause).getAllErrors());
```

Comment:
Because we get warnings about unconfirmed casts (Findbugs can't figure out that we are checking if they are of the right instance), I have refactored it this way.
I think getting warnings about unconfirmed casts is important, so I preferred to change the code rather than silence the warning.